pr-scripts/open_pr_check.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+set -o pipefail
+set -x
+
+REPO="$1"
+
+SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+
+if [ "$JOB_TYPE" != "periodic" ]; then
+    exit 0
+fi
+
+if [ $REPO_OWNER = "aws" ]; then
+    ORIGIN_ORG="eks-distro-pr-bot"
+else
+    ORIGIN_ORG=$REPO_OWNER
+fi
+
+cd ${SCRIPT_ROOT}/../../../${ORIGIN_ORG}/${REPO}
+
+PR_BRANCH="image-tag-update"
+
+ssh-agent bash -c 'ssh-add /secrets/ssh-secrets/ssh-key; ssh -o StrictHostKeyChecking=no git@github.com; git push -u origin $PR_BRANCH -f'
+
+gh auth login --with-token < /secrets/github-secrets/token
+
+PR_EXISTS=$(gh pr list | grep -c "${PR_BRANCH}" || true)
```

Comment:
I think we can check this even without pushing the branch like in the previous state. Maybe we could add the base repo as an arg to make it clear we're checking against the aws org.

Reply:
Im not totally sure what you mean here?  Above we cd into the cloned repo and the `gh pr list` looks likes it uses the current git repo/folder to figure out what repo to query.  Do you mean the `REPO_OWNER` var?

Reply:
I was also wondering how it gets the repo to query. I think it checks whether the origin repo is a fork and if it is, it queries the base repo of the fork. This is good to go.